src/Opdex.Platform.Infrastructure/Data/Handlers/Addresses/Balances/SelectAddressBalancesWithFilterQueryHandler.cs

```diff
@@ -58,7 +58,7 @@ public async Task<IEnumerable<AddressBalance>> Handle(SelectAddressBalancesWithF
         {
             var balanceId = request.Cursor.Pointer;
 
-            var queryParams = new SqlParams(balanceId, request.Address, request.Cursor.Tokens, request.Cursor.IncludeLpTokens);
+            var queryParams = new SqlParams(balanceId, request.Address, request.Cursor.Tokens, request.Cursor.TokenType == TokenProvisionalFilter.All || request.Cursor.TokenType == TokenProvisionalFilter.NonProvisional);
```

Comment:
I _think_ this should be `== TokenProvisionalFilter.NonProvisional` to set `IncludeLpTokens` to true when filtering for LP tokens. DIdn't test it but looks that way to me.

Reply:
`request.Cursor.TokenType == TokenProvisionalFilter.All || request.Cursor.TokenType == TokenProvisionalFilter.NonProvisional`

When filtering by all, includeLpTokens should be true as well

Reply:
Well, actually when filtering by all `isLpt` is unused. Guess you're right it should just be `request.Cursor.TokenType == TokenProvisionalFilter.NonProvisional`

Reply:
Ahh you mean `== TokenProvisionalFilter.Provisional`, that's right I've updated it